cmd/ingestor/main.go

```diff
@@ -365,6 +365,7 @@ func realMain(ctx *cli.Context) error {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/transfer", svc.HandleTransfer)
 	mux.HandleFunc("/readyz", svc.HandleReady)
+	mux.HandleFunc("/shutdown", svc.HandleShutdown)
```

Comment:
We can't put this on the exposed API.  You could run a denial of service attack with it.

Reply:
No longer exposing an endpoint. Thanks